Added wiki guide page in Resources => Guides => Wi-Fi Guide

abdularif0705 opened the pull request:
### Did you read and follow article requirements
yes

### What are you trying to accomplish?
add a wifi guide pg

### How are you accomplishing it?
using the links in the issue for uwin wifi

### Is there anything reviewers should know?
I'm going to cite the websites that I pulled the info from. Gotta add how to connect to uwindsor


- [x] Is it safe to rollback this change if anything goes wrong?
yes

abdularif0705 committed:
test

abdularif0705 committed:
test

abdularif0705 committed:
commit on abdul-test

abdularif0705 committed:
commit on abdul-test

prairir commented on resources/guides/wifi_guide.md:
```diff
@@ -0,0 +1,71 @@
+sidebar_label: Wi-Fi Guide # ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
+slug: /guides/wifi
+---
+# Wi-Fi Guide ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
```
This should be removed

prairir commented on resources/guides/wifi_guide.md:
```diff
@@ -0,0 +1,71 @@
+---
+id: wifi_guide
+# title: Wi-Fi Guide # ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
```
This should be uncommented and not include the image tag

kilbouri reviewed (changes requested):
resources/guides/wifi_guide.md
   Looks like a copy/paste, probably a bunch of information that doesn't apply to students that can be trimmed out.
   Also, there's a bit at the bottom about "Section breaks" that should be removed.

kilbouri commented on resources/guides/wifi_guide.md:
```diff
@@ -0,0 +1,71 @@
+
+
+
+## Section Breaks
```
Looks like is an accidental artifact of editing a template. Should be removed.

prairir replied:
this section and the `Further Reading` section arent needed

prairir commented on resources/guides/wifi_guide.md:
```diff
@@ -0,0 +1,71 @@
+![alt text](https://www.uwindsor.ca/itservices/sites/uwindsor.ca.itservices/files/246studentlaw_sm-retouched_crop.jpg "girl connecting to the right choice")
+
+**Eduroam** is a network used by many universities, and your UWindsor sign in will work at any campus that uses eduroam.
+1. Open Wi-Fi settings on your device and select the “eduroam” network
```
I think we should split it up by device or OS. for example, the instructions for linux are much different than something like windows

kilbouri replied:
+1, bump.

abdularif0705 replied:
ok I only know what to do for Windows so maybe someone else can add on for Linux

prairir replied:
Can we add a section for Iphone or android? Please add issues for platforms that you dont have. for example, an issue like `Wifi instructions for linux`

abdularif0705 replied:
ok sounds good

kilbouri commented on resources/guides/wifi_guide.md:
```diff
@@ -0,0 +1,71 @@
+
+### The IT Service Desk
+
+The IT Service Desk is located in the University Computer Centre (south entrance of the CAW building). ITS can help with any UWinID access or password issues, student email issues, and any information as to processes and policies involving University computers and networks on campus. ITS can also be reached at 519-253-3000 x4440 and is open 8:30AM to 4:30PM. Students also have the ability to repo ...
```
This seems to be false. Fact checking needed ;)

abdularif0705 committed:
Merge remote-tracking branch 'origin/abdul-test' into abdul-test

abdularif0705 committed:
cleaning up code

removing junk comments, for the wifi images

kilbouri committed:
Merge branch 'master' into abdul-test

abdularif0705 reviewed (commented):
It's showing unnecessary files that have 0 changes lmao

kilbouri commented on .github/ISSUE_TEMPLATE/bug-report.md:
```diff
@@ -1,3 +1,20 @@
+<<<<<<< HEAD
```
You need to remove these. They were added by a merge conflict and should be removed. Same goes for the `=======` and `>>>>>>> master` below.

abdularif0705 replied:
Thanks g

kilbouri commented on .github/ISSUE_TEMPLATE/code-maintenance.md:
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on .github/ISSUE_TEMPLATE/feature-request.md:
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on .github/PULL_REQUEST_TEMPLATE.md:
```diff
@@ -1,3 +1,18 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on ARCHITECTURE.md:
```diff
@@ -1,3 +1,119 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on academics/ImportantDates.md:
```diff
@@ -1,3 +1,38 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on README.md:
```diff
@@ -1,3 +1,38 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on academics/academics_sidebars.js:
```diff
@@ -1,3 +1,22 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on academics/awards/awardApplication.md:
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on babel.config.js:
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on careers/careers_sidebars.js:
```diff
@@ -1,3 +1,41 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on courses/OfferedCourses.md:
```diff
@@ -1,3 +1,76 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on courses/courses_sidebars.js:
```diff
@@ -1,3 +1,13 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on css/constitution.md:
```diff
@@ -1,3 +1,629 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on css/css_sidebars.js:
```diff
@@ -1,3 +1,13 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on docusaurus.config.js:
```diff
@@ -1,3 +1,224 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on newsletter/2021-03-17-welcome.md:
```diff
@@ -1,3 +1,21 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on package.json:
```diff
@@ -1,3 +1,40 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on resources/guides/style_guide.md:
```diff
@@ -1,3 +1,440 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on resources/resources_sidebars.js:
```diff
@@ -1,3 +1,18 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on sidebars.js:
```diff
@@ -1 +1,6 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on src/css/custom.css:
```diff
@@ -1,3 +1,112 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on src/pages/index.js:
```diff
@@ -1,3 +1,112 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on src/pages/styles.module.css:
```diff
@@ -1,3 +1,42 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

kilbouri commented on yarn.lock:
```diff
@@ -1,3 +1,10375 @@
+<<<<<<< HEAD
```
Remove these (see comment in bug-report.md)

abdularif0705 committed:
Removed unnecessary copy+paste info

Commented out some WiFi instructions that would be of use to people who are adding instructions on how to login from a Linux system

abdularif0705 committed:
Dealt with merge conflicts

Hopefully...

abdularif0705 committed:
commiting in wsl

abdularif0705 committed:
changed it to the same one from origin

abdularif0705 committed:
ran yarn prettier --write . and all file EOL's are now LF

committing thru GitKraken because it shouldn't matter anymore whether I use WSL since I ran -> git config --global core.autocrlf false

abdularif0705 committed:
got rid of the unnecessary number

abdularif0705 committed:
Big Data is a Required course for the Business & CS (dual degree) kids

Added a purple heart to it

abdularif0705 committed:
To avoid conflict I ran yarn to keep consistent with all branches

First I removed node_modules and package-lock.json in order to start with a fresh slate. Next I ran `yarn start --dev` in order to install the development dependencies.

This helps avoid conflict with other branches as npm and yarn clash when people are pulling onto their repos. Staying consistent with yarn and installing the dependencies to run prettier through yarn helps keep everything run smoothly on everyone's device.

abdularif0705 committed:
Solved the ThemeHook error that came from me running npm install

I fixed it by going back to an old commit and starting fresh from there. I used yarn to install the prettier dependencies and set the default formatter to Prettier in VSC settings

abdularif0705 committed:
Merge branch 'abdul-test' into abdul-45

abdularif0705 committed:
Solved the ThemeHook error that came from me running npm install

Fixed it by going back to an old commit and starting fresh from there. I used yarn to install the prettier dependencies and set the default formatter to Prettier in VSC settings

abdularif0705 committed:
Resolved merge conflicts

abdularif0705 committed:
Merge conflict still shows up

abdularif0705 committed:
Merge remote-tracking branch 'origin/abdul-test' into abdul-test

abdularif0705 committed:
trying to solve merge conflicts

abdularif0705 committed:
Merge remote-tracking branch 'origin/master' into abdul-test

abdularif0705 committed:
ran prettier change EOLs into LF

for some reason i ran `git config --global core.autocrlf true` but it didn't work

abdularif0705 committed:
ran prettier change EOLs into LF

for some reason i ran `git config --global core.autocrlf true` but it didn't work

abdularif0705 committed:
Merge remote-tracking branch 'origin/abdul-test' into abdul-test

prairir commented on resources/guides/wifi_guide.md:
```diff
@@ -0,0 +1,57 @@
+
+While Eduroam is the primary campus-wide Wi-Fi network, you may want to use UWinResLife in your hall for heavier use (i.e. streaming or downloading). Sign in using your UWindsor ID (just the first part - _not @uwindsor.ca_), and your UWinID password.
+
+### Activating Your UWindsor Account
```
```suggestion
## Activating Your UWindsor Account
```

prairir commented on resources/guides/wifi_guide.md:
```diff
@@ -0,0 +1,57 @@
+
+The **uwindsor** wireless network is available for use but should only be used as a **last resort**. Unlike **eduroam** and **uwinsecure**, the **uwindsor** network does not encrypt the network traffic. So, for example, if you are accessing network applications using a protocol which is insecure, (eg. http instead of https), then everyone else using the **uwindsor** network in the vicinity could  ...
+
+### [For Residence Students](https://www.uwindsor.ca/residence/wifi)
```
```suggestion
## [For Residence Students](https://www.uwindsor.ca/residence/wifi)
```

prairir reviewed (changes requested):
Overall this is great work. I just want those headers to be a bit larger

kilbouri commented on academics/awards/awardApplication.md:
```diff
@@ -10,3 +10,5 @@ Log into your [UWinsite Student][uwinsite] and complete the Award Profile. The U
 [uwinsite]: https://student.uwindsor.ca
 [guide]: https://www.uwindsor.ca/studentawards/sites/uwindsor.ca.studentawards/files/uwinaward_application_profile_instructions_for_students_3.pdf
+
+testing
```
Reason? Probably should be removed

kilbouri commented on courses/OfferedCourses.md:
```diff
@@ -53,7 +53,7 @@ hide_table_of_contents: true
 | COMP-4200   |          Mobile Application Development          | Not Offered | Offered     | Not Offered | 💛           |                       COMP-3150, COMP-3220                        |
 | COMP-4220   |            Agile Software Development            | Offered     | Not Offered | Not Offered | 💛           |                             COMP-3220                             |
-| COMP-4250   |      Big Data Analytics and Database Design      | Not Offered | Offered     | Not Offered | 💛           |                             COMP-3150                             |
+| COMP-4250   |      Big Data Analytics and Database Design      | Not Offered | Offered     | Not Offered | 💛💜         |                             COMP-3150                             |
```
Source? Just want to be sure this is correct and not a typo.

abdularif0705 replied:
I talked to my advisor about required courses and she told me that I need Big Data as it is required. And I'm part of the Business & CS program

kilbouri commented on css/constitution.md:
```diff
@@ -30,6 +30,332 @@ Society Membership shall consist of:
+### Article 2.
+
+#### Section 1 - Positions
+
```
Looks like everything above this is a duplicate of everything below. Possible residue from a merge conflict.

abdularif0705 replied:
nope, this is the exact file I copied from the master branch

kilbouri reviewed (changes requested):
I do question why all the dependency changes in `yarn.lock`. Its possible something updated, just want to know why there were so many dependencies were changed.

abdularif0705 committed:
Update README.md

abdularif0705 committed:
Update awardApplication.md

abdularif0705 committed:
Apply suggestions from code review from Ryan

Make headers larger

Co-authored-by: Ryan Prairie <me@ryanprairie.com>

prairir commented:
LGTM :rocket:

abdularif0705 commented:
> I do question why all the dependency changes in `yarn.lock`. Its possible something updated, just want to know why there were so many dependencies were changed.

no idea, all I ran was `yarn install --dev` and `yarn prettier --write .`

kilbouri reviewed (commented):
I'm confused what changed in the Constitution. The merge conflict is very confusing, an entire article seems to have been deleted.

jere-mie commented:
> I'm confused what changed in the Constitution. The merge conflict is very confusing, an entire article seems to have been deleted.

This is because we just updated the constitution to the newest version which was approved at the last CSS meeting. Whatever is currently in master should be used.

abdularif0705 committed:
Merge branch 'master' into abdul-test
